ansible/template-manifest.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: template manifest
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars:
+    SERVICE_NAME: "{{ lookup('env', 'SERVICE_NAME')}}"
+    DIST_DIR: "{{ lookup('env', 'DIST_DIR') }}"
+
+  tasks:
+    - name: set full path to manifest file
+      set_fact:
+        manifest_template: "{{ (DIST_DIR, 'manifest_template.yml') | path_join }}"
+        manifest_file: "{{ (DIST_DIR, 'manifest.yml') | path_join }}"
+
+    - name: load template streams
+      set_fact:
+        template_streams: "{{ lookup('file', manifest_template ).split('\n---\n') | list }}"
```

Comment:
might be a more yaml-aware way to do this instead of simple string split

Reply:
Apparently not, so I will leave this for now.